dockerfiles/uwsgi.ini

```diff
@@ -6,6 +7,13 @@ gid = searxng
 # Number of workers (usually CPU count)
 # default value: %k (= number of CPU core, see Dockerfile)
 workers = $(UWSGI_WORKERS)
+harakiri = 60
+max-requests = 1000                  # Restart workers after this many requests
+max-worker-lifetime = 3600           # Restart workers after this many seconds
```

Comment:
There are several open issues about requests being lost because of workers respawning at the same time. AFAIK, an undocumented parameter was added some time ago that allows smoother restarts https://github.com/unbit/uwsgi/issues/2377#issuecomment-1002560594

Reply:
@inetol thanks for pointing out, do you recommend to add "delta" and disable "strict"?


```ini
max-worker-lifetime-delta = 60
strict = false
```

Reply:
..Or removing `max-worker-lifetime*` to avoid disabling `strict`, `max-requests` *should* prevent potential memory leakage issues on its own